cpp/include/raft/random/rng.hpp

```diff
@@ -53,16 +63,16 @@ using detail::Kiss99Generator;
 template <typename Type>
 DI void box_muller_transform(Type& val1, Type& val2, Type sigma1, Type mu1, Type sigma2, Type mu2)
```

Comment:
Will be nice to see these moved into a device-specific API (and maybe even a device namespace) once we get to https://github.com/rapidsai/raft/issues/443.